### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by dzahariev**
**Description**

Changes proposed in this pull request:
- Update dependencies

**Pull Request status**
- [x] Implementation
- [x] Unit tests
- [x] Integration tests
- [x] `chart/compass/values.yaml` is updated 
- [ ] Mocks are regenerated, using the automated script

### CODE REVIEW

**Review by NickyMateev — APPROVED** [review]

**Review by DimitarPetrov — APPROVED** [review]

File: components/connector/Dockerfile [hunk 1]
```diff
@@ -13,7 +13,7 @@ RUN mkdir /app && mv ./main /app/main && mv ./licenses /app/licenses
 
 
 
-FROM alpine:3.15.0
+FROM alpine:3.15.3
```
Comment (desislavaa) [inline components/connector/Dockerfile:16]: For some reason I can't comment on L1, but bump the golang version here as well

File: components/director/Dockerfile [hunk 2]
```diff
@@ -38,7 +38,7 @@ RUN mkdir /app && mv ./director /app/director \
   && mv ./systemfetcher /app/systemfetcher \
   && mv ./licenses /app/licenses
 
-FROM alpine:3.15.0
+FROM alpine:3.15.3
```
Comment (desislavaa) [inline components/director/Dockerfile:41]: golang

File: tests/Dockerfile [hunk 3]
```diff
@@ -1,4 +1,4 @@
-FROM golang:1.17.8-alpine3.15 as builder
+FROM golang:1.18.0-alpine3.15 as builder
```
Comment (desislavaa) [inline tests/Dockerfile:1]: update it in `go.mod` as well

**Review by desislavaa — DISMISSED** [review]
As discussed offline - the adaptation for Director, Connector and tests will be in a separate PR

File: components/connector/Dockerfile (hunk 1 above)
Comment (dzahariev) [reply components/connector/Dockerfile:16]: It is not trivial and require refactoring. We agree with @NickyMateev to do this in separate task.

File: components/director/Dockerfile (hunk 2 above)
Comment (dzahariev) [reply components/director/Dockerfile:41]: It is not trivial and require refactoring. We agree with @NickyMateev to do this in separate task.

**Review by desislavaa — APPROVED** [review]

**Review by DimitarPetrov — APPROVED** [review]

**Review by NickyMateev — APPROVED** [review]

**Pull request merged**